### PR TITLE
Added markdown preview. Added round selection. Fixed homepage layout.

### LINKED ISSUE #10: Add a markdown renderer beside description input in question creation page
The question creation page has  a big text area where question description is added. Having a markdown renderer beside it would make it convenient to understand how the description would be rendered on main portal

### LINKED ISSUE #12: Implementation of round management
Implement Setround Functionality. Documents attached

https://documenter.getpostman.com/view/26244894/2sAXqtbgvt

### PR DESCRIPTION
**Pull request by abhitrueprogrammer**
Resolves #10 
Resolves #12 
- Added round selection functionality on dashboard.
- Fixed layout of dashboard to have normal sized buttons with labels on selections.
- Added markdown preview to "create question". 
- Changed to md preview from blocknote in "update question" page. 
- Fixed update question page's text to white color to maintain contrast.
image

image
image

### CODE REVIEW

**Comment by Abh1noob** [comment]
There is font inconsistency in `Add Questions` and `Edit Questions`. Fix that.

And also those text look uncentered, check about that as well

File: src/app/question/create/[qid]/page.tsx [hunk 1]
```diff
@@ -12,30 +12,26 @@ import { Input } from "@/components/ui/input";
...
 import { useEffect, useState } from "react";
-import { changeData } from "../editor";
+import Markdown from "react-markdown";
 
 const CreateButton = () => {
   const params = useParams<{ qid: string }>();
 
   const router = useRouter();
   const [question, setQuestion] = useState<QuestionResponse>();
+  const [description, setDescription] = useState<string>("teri-mummy");
```
Comment (Abh1noob) [inline src/app/question/create/[qid]/page.tsx:29]: interesting

File: src/app/question/create/page.tsx [hunk 2]
```diff
@@ -6,32 +6,28 @@ import { Input } from "@/components/ui/input";
...
-    reset,
-  } = useForm<CreateQuestionParams>();
+  const { register, handleSubmit, reset } = useForm<CreateQuestionParams>();
 
   const [sampleInputs, setSampleInputs] = useState<string[]>([""]);
   const [sampleOutputs, setSampleOutputs] = useState<string[]>([""]);
   const [explanations, setExplanations] = useState<string[]>([""]);
   const [inputFormats, setInputFormats] = useState<string[]>([""]);
-
+  const [description, setDescription] = useState<string>("teri-mummy");
```
Comment (Abh1noob) [inline src/app/question/create/page.tsx:28]: interesting 2.0

File: src/components/round.tsx
Comment (Abh1noob) [inline src/components/round.tsx:1]: add screenshot of this UI also

**Review by Abh1noob — CHANGES_REQUESTED** [review]
please do required changes

File: src/components/round.tsx
Comment (abhitrueprogrammer) [reply src/components/round.tsx:1]: It is included in the dashboard UI

File: src/app/question/create/page.tsx (hunk 2 above)
Comment (abhitrueprogrammer) [reply src/app/question/create/page.tsx:28]: Senior devs must have wrote it for a reason. Not within my discretion to change it

Comment (abhitrueprogrammer) [reply src/app/question/create/page.tsx:28]: Senior devs must have wrote it for a reason. Not within my discretion to change it

**Comment by Abh1noob** [comment]
good job

**Pull request merged**